std_msgs/Header header
BoxCandidate best_candidate
bool valid